packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -903,4 +903,25 @@
       <protected>false</protected>
       <shortDescription>Copied!</shortDescription>
     </labels>
+    <labels>
```

Comment:
@acarpentier01 Let me know if you think there is anything we can improve in the labels defined in this file. Thank you!